apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/api/Registry.kt

```diff
@@ -0,0 +1,22 @@
+package com.apollographql.apollo.gradle.api
+
+import org.gradle.api.provider.Property
+
+interface Registry {
```

Comment:
Is this class used for just grab GraphQL schema from Apollo registry? Should we add some class docs?

Reply:
Yes, it's the equivalent of `apollo client:download-schema` in the npm cli.  👍  I'll add some class docs.